packages/testing-kit/src/__TESTS__/button.spec.js

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from '@playwright/test';
+import { Button } from '../buttons/Button';  // Assuming you have this Button class
+
+test('should fire a click event and log to console', async ({ page }) => {
```

Comment:
Do these test rely on the core package? I thought it's suppose to be the other way around, no?

Reply:
we need to run storybook in core to run these tests. Playwright can start storybook by itself and run the tests.